nixos/modules/services/video/v4l2-relayd.nix

```diff
@@ -0,0 +1,197 @@
+{ config, lib, pkgs, utils, ... }:
+with lib;
+with utils;
```

Comment:
Global `with` statements are discouraged, since you are importing lots of things into the global context.

Instead you could `inherit (utils) escapeSystemdPath` to just take what you need. Same for `lib`.

Reply:
done